test/utils/mongoose-connect.js

```diff
@@ -12,7 +12,7 @@ function mongooseConnect() {
       reject(error);
     });
     db.once('open', function () {
-      resolve();
+      resolve(db);
```

Comment:
A bit dirty, I needed to access to `connection.close()` for the tests to successfully exit.